src/plib/library/View/List/Wordpress.php

```diff
@@ -56,19 +59,39 @@ private function _fetchData()
                         . ' title="' . $this->_view->escape($httpsImageTitle) . '">'
                         . ' ' . $this->_view->escape($httpsImageTitle),
                 ];
+                if (!$properties['isAlive']) {
+                    $domain = \pm_Domain::getByDomainId($wp['domainId']);
+                    $record['name'] = ' <span class="tooltipData">' . $properties['error'] . '</span>'
+                        . '<img src="' . \pm_Context::getBaseUrl() . 'images/att.png" border="0" /> '
+                        . $this->lmsg(
+                            'list.wordpress.brokenName',
+                            [
+                                'domain' => '<a href="' . PleskExt\SecurityAdvisor\Helper\Domain::getDomainOverViewUrl($domain) . '">'
+                                    . $this->_view->jsEscape($domain->getDisplayName())
+                                    . '</a>',
+                                'instance' => $record['name'],
+                            ]
+                        );
+                }
+                $wordpress[] = $record;
             }
         }
 
         return $wordpress;
     }
 
+    private static function _getDetailsUrl($id)
+    {
+        return '/modules/wp-toolkit/index.php/index/detail/id/' . $id;
```

Comment:
This link will be different on Plesk 12.5, were WordPress Toolkit is the part of Plesk core.